Meadowlark-js/lerna.json

```diff
@@ -3,12 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "v0.3.0-pre-35",
+  "version": "0.4.0-pre.0",
   "npmClient": "npm",
-  "useWorkspaces": true,
-  "command": {
```

Comment:
Although this is recommended to avoid tagging code in a different branch, in this case it is safe because the github action will not push code but only do the changes without commit